src/pages/diagram/dialogs/list-processes/components/form/form.tsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+import * as S from "./styles";
+
+interface IPayload {
+  status?: string;
+  nodeId?: string;
+  initialDate: Date | null;
+  finalDate: Date | null;
+}
+
+type Props = {
+  onClick?: (payload: IPayload) => void;
+};
+
+export const Form: React.FC<Props> = ({ onClick }) => {
+  const [searchProcess, setSearchProcess] = useState<IPayload>({
+    status: "",
+    nodeId: "",
+    initialDate: null,
+    finalDate: null,
+  });
+
+  const onChangeProcesses = (
+    value: string | Date,
+    campo: "status" | "nodeId" | "initialDate" | "finalDate"

```

Comment:
Alterar o tipo da variavel campo para keyof IPayload

```suggestion
    campo: keyof IPayload

```